hack/violation_exceptions.list

```diff
@@ -11,7 +11,6 @@ API rule violation: list_type_missing,./pkg/apis/serving/v1beta1,PodSpec,Tolerat
 API rule violation: list_type_missing,./pkg/apis/serving/v1beta1,PodSpec,Volumes
 API rule violation: list_type_missing,./pkg/apis/serving/v1beta1,PredictorConfig,SupportedFrameworks
 API rule violation: names_match,./pkg/apis/serving/v1alpha1,ModelSpec,StorageURI
-API rule violation: names_match,./pkg/apis/serving/v1beta1,AlibiExplainerSpec,StorageURI
```

Comment:
Reason to change this? Just curious what changes cause changes in this part generated.